docs/how-to/distributed-training.txt

```diff
@@ -87,11 +87,21 @@ S3. In this example, the S3 bucket name is configured via a field
 Scheduling Behavior
 -------------------
 
-To use distributed training, ``slots_per_trial`` must be set to a multiple
-of the number of GPUs per machine. For example, if you have a cluster of machines with
-8 GPUs each, you should set ``slots_per_trial`` to a multiple of 8, such as 8, 16, or 24.
-This ensures that the full network and interconnect bandwidth are available to
-multi-GPU workloads and results in better performance.
+When using distributed training, there are different scheduler constraints on the value
+``slots_per_trial`` can take depending on the default network mode and whether or not it can fit on
+a single agent.
+
+- If ``slots_per_trial`` is smaller than the number of slots on any available agents and
```

Comment:
Yeah, I agree its confusing, splitting it out like above makes it a little easier to understand. I'll make that change.

> Also, when we are talking about the relationship between slots_per_trial and the # of slots on any agent -- suppose I have one agent with 16 GPUs and 10 agents with 8 GPUs each. If I submit an 8 GPU job, is it possible that it gets spread over multiple agents?

No, in that case is should only ever get scheduled onto a single agent. If an experiment is spread over multiple agents, it will always take up entire agents.

Reply:
@neilconway what do you think now?

Reply:
@stoksc I made some edits and pushed to the branch -- lmk what you think.

Reply:
It looks good to me now. I think I approach docs trying to rewrite (change) as little as possible and I think it's leading my astray; these look much better and they're mostly rewritten. I'll let @aaron276h take the last pass tomorrow, though, but I would approve this.